# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1302681721 -7200
#      Wed Apr 13 10:02:01 2011 +0200
# Node ID ca927002fd98e6c16f737f6a0e8ec3ff03473fb4
# Parent  935cc330f4ab71e8232bc8a059ab984d1c5613a8
Tab focus switches to editor when files has been opened.

diff --git a/gui//src/MainWindow.cpp b/gui//src/MainWindow.cpp
--- a/gui//src/MainWindow.cpp
+++ b/gui//src/MainWindow.cpp
@@ -37,16 +37,17 @@ MainWindow::~MainWindow() {
 }
 
 void MainWindow::handleOpenFileRequest(QString fileName) {
     reportStatusMessage("Opening file.");
     FileEditorMdiSubWindow *subWindow = new FileEditorMdiSubWindow(m_openedFiles);
     m_openedFiles->addSubWindow(subWindow);
     subWindow->loadFile(fileName);
     subWindow->showMaximized();
+    m_centralTabWidget->setCurrentWidget(m_openedFiles);
 }
 
 void MainWindow::reportStatusMessage(QString statusMessage) {
     m_statusBar->showMessage(statusMessage, 1000);
 }
 
 void MainWindow::openWebPage(QString url) {
     m_browserWidget->load(QUrl(url));
